src/main/webapp/js/captions.js

```diff
@@ -4,8 +4,8 @@
  * execute                  -- execute a request to download captions from Youtube API
  * successfulDisplay
  * displayVideo
- * beginCaptionRequest
- * getCaptions
+ * getTrackId
+ * getYTCaptions
  * parseCaptionsIntoJson
  * getIdFromUrl
  * sendJsonForm
```

Comment:
just double checking we made sure to update the function list here

Reply:
I updated all mentions of the  functions, including these comments